### PR TITLE
chore: update changelogs for main w/ v3.0.2 content

### PR DESCRIPTION
**Pull request by traeok**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds or improves functionality)
- [ ] Breaking change (a change that would cause existing functionality to not work as expected)
- [x] Documentation (Markdown, ~README~ CHANGELOG updates)
- [ ] Other (please specify above in "Proposed changes" section)

### CODE REVIEW

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
-- Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish. Download the respective language pack and switch.
-- Localization of strings within the webviews. [#2983](https://github.com/zowe/zowe-explorer-vscode/issues/2983)
-- Added expired JSON web token detection for profiles in each tree view (Data Sets, USS, Jobs). When a user performs a search on a profile, they are prompted to log in if their token expired. [#3175](https://github.com/zowe/zowe-explorer-vscode/issues/3175)
```
Comment (JillieBeanSim) [inline packages/zowe-explorer/CHANGELOG.md:11]: should this item stay for 3.1 under TBD for now?

File: packages/zowe-explorer-api/CHANGELOG.md [hunk 2]
```diff
@@ -6,12 +6,15 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
-- Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish.
-- Localization of strings within the webviews. [#2983](https://github.com/zowe/zowe-explorer-vscode/issues/2983)
-- Update Zowe SDKs to `8.2.0` to get the latest enhancements from Imperative.
```
Comment (JillieBeanSim) [inline packages/zowe-explorer-api/CHANGELOG.md:11]: This item should also stay under the TBD

**Review by JillieBeanSim — CHANGES_REQUESTED** [review]
Thanks @traeok for putting the port PR together. left a couple comments

File: packages/zowe-explorer/CHANGELOG.md (hunk 1 above)
Comment (traeok) [reply packages/zowe-explorer/CHANGELOG.md:11]: For this PR, I did a `git checkout` of the changelogs on the v3.0.2 tag. The contents should match the changelogs on 3.0.2, but with the TBD release headers added above:

https://github.com/zowe/zowe-explorer-vscode/blob/v3.0.2/packages/zowe-explorer/CHANGELOG.md#302

Comment (traeok) [reply packages/zowe-explorer/CHANGELOG.md:11]: Should be resolved in 7f67562 👍

**Review by JillieBeanSim — APPROVED** [review]

**Pull request merged**